sections/plate_notation.tex

```diff
@@ -0,0 +1,61 @@
+\section{Our Topic Models}\label{sec:plate_notation}\vejleder{Proposed Topic Models, eller rename til noget mere fornemt}
+In this section, we detail the modifications that we have implemented to describe our proposed topic models, outlined in \autoref{sec:preliminaries}.
+We present three models, one for each of the metadata fields detailed in \autoref{sec:dataset}.
+Two of the models: the Author and Category metadata models are based on the Author-topic \gls{lda} explained in \autoref{subsec:auth_prelim}.
+Lastly the Taxonomy metadata model is based on \gls{pam}, which is explained in \autoref{subsec:pachinko_prelim}.
+
+\subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
+We model both of the metadata fields 'Author' and 'Category' similarly to the model by \citet{author_topic_2012} presented in \autoref{subsec:auth_prelim}.
+The category model builds on the assumption that categories of the articles were chosen based on the content of the article and that local newspapers have their own unique topic preferences \vejleder{hvordan afspejles det i modellerne?}.
+We find this model to be generally applicable to most metadata information, assuming that the metadata is either chosen based on the text of the documents\vejleder{hvad med det?} (as with the category metadata) or that it has some impact on the text of the document (as with the author metadata).
+
+For our category-topic model and our author-topic model, each document $d$ is associated with only one category $c_d$ from a set of categories $C$ and only one author $a_d$ from a set of authors $A$, rather than a vector.
```

Comment:
```suggestion
For our category-topic model and our author-topic model, each document $d$ is associated with only one category $c_d$ from a set of categories $C$ and only one author $a_d$ from a set of authors $A$, rather than a vector, respectively.
```